gui/qt/paytoedit.py

```diff
@@ -266,23 +266,24 @@ def textUnderCursor(self):
         return tc.selectedText().strip()
 
     def keyPressEvent(self, e):
-        if self.isReadOnly():
+        if self.isReadOnly() or not self.hasFocus():
+            e.ignore()
             return
 
         if self.c.popup().isVisible():
             if e.key() in [Qt.Key_Enter, Qt.Key_Return]:
                 e.ignore()
                 return
 
-        if e.key() in [Qt.Key_Tab]:
+        if e.key() in [Qt.Key_Tab, Qt.Key_Backtab]:
             e.ignore()
             return
 
         if e.key() in [Qt.Key_Down, Qt.Key_Up] and not self.is_multiline():
             e.ignore()
             return
 
-        QPlainTextEdit.keyPressEvent(self, e)
+        super().keyPressEvent(e)
```

Comment:
lol. Yeah. 👍 

Esp. now that we do base class magic here.  Good catch.

Reply:
Also -- do we want to check if e.isIgnored() here (or whatever the method is called in Qt)? Or? I forget what the good well-behaved contract should be. Just asking...

Reply:
isAccepted would always be true, it is the default.

Reply:
ah heh

ok nevermind.. will just merge